RE: Sev 3: Id 54668536: F series VM's having inconsistent CPU's "Xeon…

axelgMS opened the pull request:
… E5-2673 v3" and "Xeon E5-2673 v4"

RE: Sev 3: Id 54668536: F series VM's having inconsistent CPU's "Xeon E5-2673 v3" and "Xeon E5-2673 v4"

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @mahesh-unnikrishnan, has been notified to review your proposed change.

mahesh-unnikrishnan commented:
Thanks for bringing this to our notice. I've resolved this typo as part of a recent doc update.

4c74356b41 commented:
same question, should I close it?

mahesh-unnikrishnan commented:
Please go ahead and close this PR.

axelgMS committed:
RE: Sev 3: Id 54668536: F series VM's having inconsistent CPU's "Xeon E5-2673 v3" and "Xeon E5-2673 v4"

RE: Sev 3: Id 54668536: F series VM's having inconsistent CPU's "Xeon E5-2673 v3" and "Xeon E5-2673 v4"
